dmscripts/helpers/supplier_data_helpers.py

```diff
@@ -155,3 +158,15 @@ def get_supplier_ids_from_file(supplier_id_file):
         return None
     with open(supplier_id_file, 'r') as f:
         return list(map(int, [_f for _f in [l.strip() for l in f.readlines()] if _f]))
+
+
+@lru_cache()
+def country_code_to_name(country_code):
+    register, code = country_code.split(':')
+
+    register_response = requests.get(f'https://{register}.register.gov.uk/records/{code}.json')
+    if register_response.status_code == 200:
```

Comment:
Would we not want to raise an exception here if the status code isn't 200? Could sliently end up with a mix of country codes and names in the output from the script?

Could maybe with adding backoff to retry in case of none 200s?

Reply:
Yep, done.